src/Compilers/Core/Portable/Symbols/ISymbol.cs

```diff
@@ -58,6 +58,11 @@ public interface ISymbol : IEquatable<ISymbol?>
         /// </summary>
         string MetadataName { get; }
 
+        /// <summary>
+        /// Gets the metadata token
+        /// </summary>
+        int MetadataToken { get; }
```

Comment:
Compiler public APIs need direct testing

Reply:
Yeah, none of the files in that folder do, so I maintained the pattern. Happy to rename it if preferred (I also couldn't find where to put these tests because of the same issue!)